### ISSUE TITLE
Payment for REV jobs should be based on the number of issues found

### ISSUE THREAD

**Issue by vzurauskas** [zerocracy/farm#2100]
At the moment code reviewer gets full payment if he finds three or more non-cosmetic issues. This gives me a very strong incentive to stop reviewing after I comment on three non-cosmetic issues. This also gives me an incentive not to review code changes the DEV commits addressing my issues and just approve anything.

I think the payment should be based on issues found, perhaps 5 mins per issue? So if I find 3 issues, I get paid the same amount as currently, but that gives me an incentive to continue reviewing and finding more issues. I know I will not get overboard because the more meaningless issues I find, the more likely DEV is going to argue with me, and we will be wasting each other's time. So my incentive would be to quickly find real issues, such that DEV will not be arguing with me, and move to other PRs.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@vzurauskas[/z](https://www.0crat.com/u/vzurauskas) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@vzurauskas I like this idea, you suggested that QA needs to say amount of issues in the verdict, right? E.g. `quality good, pr.issues=5`, or something.

**Comment by g4s8** [comment]
@yegor256 please check this

**Comment by vzurauskas** [comment]
@g4s8 Yes, something like that.

**Comment by yegor256** [comment]
@g4s8 a few years ago we had a different formula. We were paying one extra minute per each comment posted by the reviewer in the pull request. The longer the discussion, the larger the payment. I still think it's a good idea. WDYT?

**Comment by g4s8** [comment]
@yegor256 yes, I think it should work. @vzurauskas see https://github.com/zerocracy/farm/issues/2100#issuecomment-508466905

**Comment by 0crat** [comment]
Job [#2100](https://github.com/zerocracy/farm/issues/2100) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @vzurauskas[/z](https://www.0crat.com/u/vzurauskas)

**Comment by vzurauskas** [comment]
@yegor256 @g4s8 I think that's a good idea. Sometimes a long discussion can be useful, it makes both REV and DEV think about how the code can be improved, even if it doesn't always result in actual improvement then and there. However, I think there should be some way to prevent REV from abusing this rule by intentionally spamming the PR with comments. Maybe QA should just use common sense?
